docs/conf.py

```diff
@@ -210,10 +212,33 @@
 """
 
 
+object_description_options_map["std:confval"] = dict(
+    toc_icon_class="data", toc_icon_text="C", generate_synopses="first_sentence"
+)
+
+object_description_options_map["std:objconf"] = dict(
+    toc_icon_class="data", toc_icon_text="O", generate_synopses="first_sentence"
+)
+
+
+def _validate_parallel_build(app):
+    # Verifies that all of the extensions defined by this theme support parallel
+    # building.
+    assert app.is_parallel_allowed("read")
+    assert app.is_parallel_allowed("write")
```

Comment:
This feels like a "slap on the wrist". 🤣 My bad.

Reply:
It is easy to miss since we had no test, and I also missed it when reviewing.  I only notice when building the TensorStore docs since they are large and slow to build without parallel building.